scripts/docker

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
-IMAGE=NodeOS
-USERSFS=out/latest/usersfs
+IMAGE=nodeos
+USERSFS=/out/latest/usersfs
```

Comment:
i dont know but thats the only time i dont get a validation error like this: 
```bash
docker: Error response from daemon: create out/latest/usersfs: "out/latest/usersfs" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed.
See 'docker run --help'.
```

Or if the path is relative i get a error that says that docker needs a absolute path